TACTLib/Core/Product/Odin/OdinProductHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.IO;
+using TACTLib.Client;
+
+namespace TACTLib.Core.Product.Odin {
+	[ProductHandler(TACTProduct.ModernWarfare)]
+	public class OdinProductHandler : IProductHandler {
```

Comment:
as i said before, wot is the point in this class.

Reply:
I don't really want to expose `VFSFilesTree::_files`, I would rather have it exposed from the `IProductHandler` responsible for it, also because TACTLib isn't as useful without a program using it, I don't know if there is more to be done and I plan to start working on the basics of that program to see the ideal data structures I should expose, hence why this PR is marked as DNM at the moment.

Reply:
pro tip, u can make draft PRs

Reply:
code for nicely handling the vfs should be done in a product independant way